modules/enabling-scanner-v4-after-helm-installation.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-scanner-v4-after-helm-installation_{context}"]
+= Enabling RHACS Scanner V4 after Helm installation
+
+[role="_abstract"]
+Scanner V4 is not enabled by default, but you can enable Scanner V4 after installation.
+
+If you want to enable {product-title-short} on secured clusters by using Helm charts, you must first enable Scanner V4 on the cluster where Central is installed.
+
+.Procedure
+
+. Create a `values-public.yaml` file that contains the following lines:
++
+[source,yaml]
+----
+scannerV4:
+    disable: false
+----
```

Comment:
Mentioned in other comment, consider for the helm instructions using the `--set` CLI argument for consistency (or updating everywhere to use values file)